config/localization/en/tooltips.json

```diff
@@ -290,6 +290,18 @@
       "TITLE": "Fast Withdraw Fee",
       "BODY": "Fast withdraws are sent immediately and dYdX will cover the gas costs in exchange for a fee. For fee-free withdraws, please use a slow withdraw (you pay your own gas costs and will have to wait for Layer 2 confirmation)."
     },
+    "GAS_FEES":{
+      "BODY": "Gas fees are transaction fees on the source chain, in this case dYdX"
+    },
+    "SUBHEADER_PRE_DYNAMIC_TEXT": {
```

Comment:
I think this will cause issues with internationalization (different languages don't always have the same sentence structure that allow us to split translations into the pre + post like this) - what's going in between pre and post? I think you could try using a parameter like [here](https://github.com/dydxprotocol/v4-localization/blob/main/config/localization/en/tooltips.json#L4)

Reply:
amazing. i probably should have asked instead of assuming this was impossible. thanks a ton!